### PR TITLE
Improve docs about reraise

### PR DESCRIPTION
**Pull request by dsyme**
Alex Zeit mentioned that the docs for `reraise` were a little lacking.

This improves somewhat.

### CODE REVIEW

File: docs/fsharp/language-reference/exception-handling/the-raise-function.md [hunk 1]
```diff
@@ -27,6 +28,16 @@ The `raise` function can also be used to raise .NET exceptions, as shown in the
 
 [!code-fsharp[Main](~/samples/snippets/fsharp/lang-ref-2/snippet5802.fs)]
 
+## Reraising an exception
+
+The `reraise` function can be used in a `with` block to propagate a handled exception up the call chain.
+Note that `reraise` does not take an exception operand. It is most useful when a method passes on an
```
Comment (gewarren) [inline docs/fsharp/language-reference/exception-handling/the-raise-function.md:34]:
```suggestion
`reraise` does not take an exception operand. It's most useful when a method passes on an
```

Acrolinx, our grammar checker, would ask us to remove that.

**Review by gewarren — APPROVED** [review]
LGTM. Left some style guide suggestions.

File: docs/fsharp/language-reference/exception-handling/the-raise-function.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
 ---
-title: "Exceptions: the raise Function"
-description: Learn how the F# 'raise' function is used to indicate that an error or exceptional condition has occurred.
+title: "Exceptions: The raise and reraise functions"
+description: Learn how the F# 'raise' and 'reraise' functions are used to indicate that an error or exceptional condition has occurred.
 ms.date: 05/16/2016
 ---
 # Exceptions: the raise Function
```
Comment (luisquintanilla) [inline docs/fsharp/language-reference/exception-handling/the-raise-function.md:6]:
```suggestion
# Exceptions: raise and reraise functions
```

File: docs/fsharp/language-reference/exception-handling/the-raise-function.md [hunk 3]
```diff
@@ -1,11 +1,12 @@
 ---
-title: "Exceptions: the raise Function"
-description: Learn how the F# 'raise' function is used to indicate that an error or exceptional condition has occurred.
+title: "Exceptions: The raise and reraise functions"
+description: Learn how the F# 'raise' and 'reraise' functions are used to indicate that an error or exceptional condition has occurred.
 ms.date: 05/16/2016
```
Comment (luisquintanilla) [inline docs/fsharp/language-reference/exception-handling/the-raise-function.md:4]:
```suggestion
ms.date: 07/23/2021
```

File: docs/fsharp/language-reference/exception-handling/the-raise-function.md [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
 ---
 # Exceptions: the raise Function
 
 The `raise` function is used to indicate that an error or exceptional condition has occurred. Information about the error is captured in an exception object.
+The `reraise` function is used to propagate a handled exception up the call chain.
```
Comment (luisquintanilla) [inline docs/fsharp/language-reference/exception-handling/the-raise-function.md:9]: Adding bullet points to stress they're different 

```suggestion
- The `raise` function is used to indicate that an error or exceptional condition has occurred. Information about the error is captured in an exception object.
- The `reraise` function is used to propagate a handled exception up the call chain.
```

File: docs/fsharp/language-reference/exception-handling/the-raise-function.md [hunk 5]
```diff
@@ -1,11 +1,12 @@
 ---
-title: "Exceptions: the raise Function"
-description: Learn how the F# 'raise' function is used to indicate that an error or exceptional condition has occurred.
+title: "Exceptions: The raise and reraise functions"
```
Comment (luisquintanilla) [inline docs/fsharp/language-reference/exception-handling/the-raise-function.md:2]:
```suggestion
title: "Exceptions: raise and reraise functions"
```

**Review by luisquintanilla — COMMENTED** [review]
Looks good. Added minor updates for formatting & making intent clearer

**Pull request merged**